src/pages/OnboardingEmployees/BaseOnboardingEmployees.tsx

```diff
@@ -82,9 +81,7 @@ function BaseOnboardingEmployees({shouldUseNativeStyles, route}: BaseOnboardingE
                 onBackButtonPress={OnboardingFlow.goBack}
             />
             <View style={[onboardingIsMediumOrLargerScreenWidth && styles.mt5, onboardingIsMediumOrLargerScreenWidth ? styles.mh8 : styles.mh5]}>
-                <View style={[onboardingIsMediumOrLargerScreenWidth ? styles.flexRow : styles.flexColumn, styles.mb5]}>
-                    <Text style={[styles.textHeadlineH1]}>{translate('onboarding.employees.title')}</Text>
-                </View>
+                <Text style={[styles.textHeadlineH1, styles.mb5]}>{translate('onboarding.employees.title')}</Text>
             </View>
```

Comment:
Do we still need this outer View?

(Same on BaseOnboardingAccounting)

Reply:
Removed.